core/src/labels/label.cpp

```diff
@@ -132,8 +131,11 @@ void Label::enterState(State _state, float _alpha) {
 }
 
 void Label::setAlpha(float _alpha) {
-    m_transform.state.alpha = CLAMP(_alpha, 0.0, 1.0);
-    m_dirty = true;
+    float alpha = CLAMP(_alpha, 0.0, 1.0);
+    if (m_transform.state.alpha != alpha) {
```

Comment:
I mean just the test equality to 0.0 or 1.0 - while the alpha is in between it is most likely to change each frame. 
The screenshot is missing the value of m_transform.state.alpha to be meaningful :)